examples/rust/_examples_common/custom_header.rs

```diff
@@ -10,7 +10,10 @@
 //
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-#[derive(Default, Debug)]
+use iceoryx2_bb_derive_macros::ZeroCopySend;
+
+#[derive(Default, Debug, ZeroCopySend)]
+#[type_name(CustomHeader)]
```

Comment:
Can you add a comment at the end of this line that this is optional and explain that when this is not set `core::any::type_name::<T>()` will be used.